Source/SwiftLintFramework/Rules/MissingDocsRule.swift

```diff
@@ -155,6 +155,10 @@ public struct MissingDocsRule: OptInRule {
     )
 
     public func validate(file: File) -> [StyleViolation] {
+        guard SwiftVersion.current == .two else {
+            warnMissingDocsRuleDisabledOnce
```

Comment:
I don't think it's bad, but I definitely took a while to understand what's going on. I didn't know it was in a fixit, so maybe it'll become more popular.